deisctl/client/client.go

```diff
@@ -240,6 +241,20 @@ Usage:
 	return cmd.ListUnits(c.Backend)
 }
 
+func (c *Client) Machines(argv []string) error {
+	usage := `Enumerate the current hosts in the cluster
```

Comment:
We were just copying the fleetctl list-machines cmd description here, but yes list works too.